### PR TITLE
Intro to OCS Updates

### PR DESCRIPTION
**Pull request by matthewcharles27**
Moved in TOC, edits to text, image updates.

### CODE REVIEW

File: content/overview/get-started/view-profile.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ The **Show user friendly property names** option controls how property names app
 
 To change your user preferences:
 
-1. To open your user profile, select the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
+1. Click the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name in the drop-down.
```
[the remark this answers is not in the document]
Comment (afloresOSI) [reply content/overview/get-started/view-profile.md:30]: If you want to leave in the last phrase, it should be "in the dropdown list."

**Review by afloresOSI — CHANGES_REQUESTED** [review]

File: content/overview/get-started/view-profile.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ uid: viewProfile
 # View and edit your user profile
 
 The `My Profile` window displays your user information and assigned roles.
-Depending on the identity provider your organization uses, you may be able to update your contact information.
+Depending on the Identity Provider your organization uses, you may be able to update your contact information.
```
Comment (jasnames) [inline content/overview/get-started/view-profile.md:8]: While there is an Identity Providers page in OCS (not ADH), identity provider itself isn't a proper noun and isn't capitalized elsewhere.

File: content/overview/get-started/view-profile.md [hunk 3]
```diff
@@ -5,19 +5,19 @@ uid: viewProfile
...
 
 To update your contact information:
 
-1. To open your user profile, select the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
+1. To open your user profile, click the **User Profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
 
 1. Select **Edit**.
 
-    **Note**: If the Identity Provider your organization uses does not allow you to update your contact information, the **Edit** button is disabled.
+    **Note:** The **Edit** button is disabled if your organization's Identity Provider does not allow you to update your contact information.
```
Comment (jasnames) [inline content/overview/get-started/view-profile.md:18]:
```suggestion
    **Note:** The **Edit** button is disabled if your organization's identity provider does not allow you to update your contact information.
```

**Review by jasnames — CHANGES_REQUESTED** [review]
Not part of your changes, but also remove "The **Information** tab appears in some windows and provides documentation about the current feature and how to perform relevant tasks" from introduction-portal-interface.md. Info tabs are removed, or being removed, from the product.

File: content/overview/get-started/view-profile.md [hunk 4]
```diff
@@ -5,19 +5,19 @@ uid: viewProfile
...
 The `My Profile` window displays your user information and assigned roles.
-Depending on the identity provider your organization uses, you may be able to update your contact information.
+Depending on the Identity Provider your organization uses, you may be able to update your contact information.
 
 ## Update contact information
 
 To update your contact information:
 
-1. To open your user profile, select the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
+1. To open your user profile, click the **User Profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
```
[the remark this answers is not in the document]
Comment (matthewcharles27) [reply content/overview/get-started/view-profile.md:14]: Done.

File: content/overview/get-started/view-profile.md [hunk 5]
```diff
@@ -27,6 +27,6 @@ The **Show user friendly property names** option controls how property names app
 
 To change your user preferences:
 
-1. To open your user profile, select the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name.
+1. Click the **User profile** icon ![User Profile icon](../../_icons/custom/account-circle.svg) and select your name in the drop-down.
 
-1. To use friendly property names, turn on the the toggle under **Show user friendly property names**. The setting is saved automatically.
+1. Turn on the the **Show user friendly property names** toggle under **User Preferences**. The setting saves automatically.
```
[the remark this answers is not in the document]
Comment (matthewcharles27) [reply content/overview/get-started/view-profile.md:32]: Put "To use friendly property names..." back into the sentence.

File: content/overview/get-started/view-profile.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthewcharles27) [reply content/overview/get-started/view-profile.md:30]: Changed to "To open your user profile, select the **User profile** icon User Profile icon and select your name from the dropdown list."

Changed in to from in the last bit; I think it sounds a little better that way.

File: content/overview/get-started/view-profile.md (hunk 2 above)
Comment (matthewcharles27) [reply content/overview/get-started/view-profile.md:8]: Done.

File: content/overview/get-started/view-profile.md (hunk 3 above)
Comment (matthewcharles27) [reply content/overview/get-started/view-profile.md:18]: Done

**Comment by matthewcharles27** [comment]
> Not part of your changes, but also remove "The **Information** tab appears in some windows and provides documentation about the current feature and how to perform relevant tasks" from introduction-portal-interface.md. Info tabs are removed, or being removed, from the product.

Thanks, Jason. Since removing the information tab bit left the bullet points looking funny, I got rid of them entirely and changed the section to this: "The right pane may contain a **Details** tab that contains additional information about the selected resource. Some resources also have additional panes for other configuration tasks."

**Review by afloresOSI — APPROVED** [review]

File: content/home/health.md [hunk 6]
```diff
@@ -4,19 +4,19 @@ uid: home-health
 
 # Health
 
-The **Health** page displays the health status of each service and namespace in your OSIsoft Cloud Services (OCS) tenant. You can access this page from the **Home** page by selecting **System Health**.
+The **Health** page displays the health status of each service and namespace in your OSIsoft Cloud Services (OCS) tenant. Select **System Health** from the **Home** page to view your system health status.
```
Comment (jasnames) [inline content/home/health.md:7]:
```suggestion
The **Health** page displays the health status of each service and namespace in your OSIsoft Cloud Services (OCS) tenant. To view your system health status, select **System Health** from the **Home** page.
```

**Review by jasnames — COMMENTED** [review]
One last suggestion, going along with Angela's earlier comment about putting what you are doing before the action.

**Pull request merged**